libraries/helpers_firewalld.rb

```diff
@@ -39,6 +39,8 @@ def firewalld_flush!
 
         shell_out!('firewall-cmd', '--direct', '--remove-rules', 'ipv4', 'filter', 'INPUT')
         shell_out!('firewall-cmd', '--direct', '--remove-rules', 'ipv4', 'filter', 'OUTPUT')
+        shell_out!('firewall-cmd', '--direct', '--remove-rules', 'ipv6', 'filter', 'INPUT')
```

Comment:
Could you add a guard for `ipv6_enabled?(new_resource)`? Just so we don't run this if ipv6 is disabled? Or confirm that it won't blow up if ipv6 isn't present/enabled on the system?